Makefile.devel

```diff
@@ -89,14 +89,14 @@ DEVELOPERS=\
 # PRERELEASE_PREFIX is used to match PRERELEASE_TAG in sed
 # line in tag target below, which edits $(DOWNLOADHTML)
 # The prereltag.txt is kept as a record in the distrib area
-# of the current pre-release version (currently not used explicitly 
+# of the current pre-release version (currently not used explicitly
 # anywhere for web pages/whatever).
-PRERELEASE_PREFIX=4\.4\.1~pre
-PRERELEASE_TAG=4.4.1~pre
+PRERELEASE_PREFIX=4\.4\.1-pre
```

Comment:
I certainly can: it's just text. :-)  However, isn't this project ultimately producing an Emacs package? If so, I submit that the relevant guide lines with respect to version numbers are  [the guide lines for Emacs package version numbers](https://www.gnu.org/software/emacs/manual/html_node/elisp/Packaging-Basics.html#Packaging-Basics).  My reading of those guide lines does not lead me to believe that "~" will "out of the box"  as part of an Emacs package version string ("out of the box" = "without having the user customize `version-separator` and/or `version-regexp-alist`").